.github/workflows/build.yaml

```
@@ -268,7 +270,7 @@ jobs:
       run: |
         mkdir build_wl
         cd build_wl/
-        ../utils/macos/build_app.sh ${{ matrix.compiler }} ${{ matrix.config }} ../
+        ../utils/macos/build_app.sh ${{ matrix.config }} ${{ matrix.compiler }}
```

```suggestion
        ../utils/macos/build_app.sh "--${{ matrix.config }}" "--${{ matrix.compiler }}"
```
Then you can revert the changes to the matrix definition above